src/webui/prefjson.cpp

```diff
@@ -347,7 +347,7 @@ void prefjson::setPreferences(const QString& json)
         pref->setIgnoreSlowTorrentsForQueueing(m["dont_count_slow_torrents"].toBool());
     // Share Ratio Limiting
     if (m.contains("max_ratio_enabled"))
-        pref->setGlobalMaxRatio(m["max_ratio"].toInt());
+        pref->setGlobalMaxRatio(m["max_ratio"].toDouble());
```

Comment:
I saw this => https://github.com/qbittorrent/qBittorrent/blob/a19e2e48a15766b468acb12903d1868c0198cd2b/src/base/preferences.cpp#L972
I can change both if you want.